.changeset/mighty-lions-search-2.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-test-utils': minor
+---
+
+**BREAKING**: Renamed `mockServices.config` to `mockServices.rootConfig`.
```

Comment:
Realized this rename was missing so threw that in too. Also split the changesets a bit more